### PR TITLE
Update readme and changelog.

### PR DESCRIPTION
**Pull request by maethu**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -156,6 +156,11 @@ provided to jump to the original content.
 The selectable blocks are returned by
 ``ftw.simplelayout.aliasblock.contents.aliasblock.get_selectable_blocks``.
 
+It's also possible to select a whole ContentPage to render within the AliasBlock.
+It's not possible to select another ContentPage which already contains an AliasBlock.
+Alle Blocks within this package are supported (incl. MapBlock). ftw.sliderblock and ftw.addressblock are working too.
+Other Blocks may work, but they are not tested.
```
Comment (Nachtalb) [inline README.rst:162]: I'd make it a bit shorter and remove unnecessary information.

```suggestion
It's also possible to select a whole ContentPage to render within the AliasBlock unless that page already contains another AliasBlock (prevent recursion).
All blocks within this package are supported (incl. MapBlock).
Blocks from other packages may work but are not tested.
```

Comment (djowett-ftw) [inline README.rst:162]: As commented elsewhere - Blocks from other packages won't work because the list of aliasable blocks is hard coded

Comment (Nachtalb) [reply README.rst:162]: cood catch 👍

**Comment by maethu** [comment]
@djowett-ftw  @Nachtalb updated.

**Pull request merged**